nvflare/private/fed/client/fed_client_base.py

```diff
@@ -190,7 +190,8 @@ def fetch_execute_task(self, project_name, fl_ctx: FLContext):
 
             return task
         except FLCommunicationError as e:
-            self.logger.info(e)
+            if e:
+                self.logger.info(e)
```

Comment:
Since this exception is thrown, there must be something in there.

We should look for how we construct this `FLCommunicationError` and put relevant information there?

Reply:
Added the FLCommunicationError for different places.

Reply:
thanks, then we should just log them out right?
so we don't need this "if" condition

Reply:
done